shopping_mall/backend/app/services/ai_chatbot.py

```
@@ -101,12 +117,20 @@ async def _handle_delivery(self, db: Session, question: str, user_id: int | None
 
                 if self.llm:
                     try:
+                        history_text = ""
+                        if history:
```

Ruff (0.15.9)

[error] Undefined name `history`

(F821)